packages/medusa/src/loaders/helpers/routing/__fixtures__/server/index.js

```diff
@@ -88,10 +84,6 @@ export const createServer = async (rootDir) => {
   })
 
   featureFlagLoader(config)
-  modelsLoader({ container, isTest: true })
-  repositoriesLoader({ container, isTest: true })
-  servicesLoader({ container, configModule: config })
-  strategiesLoader({ container, configModule: config })
   await passportLoader({ app: app, container, configModule: config })
```

Comment:
do we still use passport? I think we can can this as well

Reply:
@riqwan I left some things I wasn't 100% sure of, but I think we don't. I will do a follow-up PR with the smaller cleanups, wanted to get the bulk out of the way.